src/server/worker/master.go

```diff
@@ -117,31 +97,8 @@ func (a *APIServer) serviceMaster() {
 			return err
 		}
 		defer masterLock.Unlock(ctx)
-
 		logger.Logf("Launching master process")
```

Comment:
One thing I might want to change: we don't have a way to log which spawner this master process is using, which might be a good thing to log for debugging purposes. I think the simplest and most robust way would just be to add a `spawnerName` field to the master function. Trying to hacky things to get the function name directly from the spawner pointer might also be possible, but seems like it would add undue complexity here.